### PR TITLE
Add WNP for EU (fixes #12230)

### LINKED ISSUE #12230: WNP 107 (eu)
## Description

Design: [here](https://www.figma.com/file/8fVLbzzAd40lSSLuOwCfnl/EU_Firefox_WNP_21-22?node-id=2138%3A400)

Copy: [here](https://docs.google.com/document/d/1RWZoDMGpdl5RlZnUMAFAGvDN33sd9SR5yZTqb3W6Nao/edit#)

Adjust Links: TBD (@slightlyoffbeat  to deliver)

---

#### Success Criteria
- [ ] Test and confirm android attribution
- [ ] Test and confirm iOS attribution

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Adds WNP107 templates for DE and FR

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12230

## Checklist

- [ ] Tests added

## Testing

To test this work:

- [ ] http://localhost:8000/fr/firefox/107.0/whatsnew/
- [ ] http://localhost:8000/de/firefox/107.0/whatsnew/

⚠️ waiting on Firefox Mobile Adjust links
~🟢 Adjust links available!~ whoops, wrong product

### CODE REVIEW

**Comment by maureenlholland** [comment]
ready for code review, will drop in Adjust links when available

File: media/css/firefox/whatsnew/whatsnew-107-eu.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import 'includes/base';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+
+.wrapper {
+    background-color: $color-violet-50;
```
Comment (alexgibson) [inline media/css/firefox/whatsnew/whatsnew-107-eu.scss:17]: Issue (non-blocking): the background and body copy colour does pass contrast checks ok, but honestly I find it pretty hard to read still (maybe it's just me?)

Screenshot 2022-11-11 at 10 04 01

**Review by alexgibson — CHANGES_REQUESTED** [review]
One blocking issue with the adjust links sending to the wrong product, and a borderline a11y contrast issue (non-blocking). But otherwise everything else looks good :)

File: media/css/firefox/whatsnew/whatsnew-107-eu.scss (hunk 1 above)
Comment (maureenlholland) [reply media/css/firefox/whatsnew/whatsnew-107-eu.scss:17]: Not just you, I was surprised it passed color contrast too, will look into a slight adjustment (maybe lighter BG color? the larger heading font should still be ok)

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx107-de.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  <div class="wnp-main-content mzp-l-content mzp-t-content-xl mzp-l-columns mzp-t-columns-two">
+    <div>
+      <h2 class="wnp-main-title">Mach da weiter, wo du aufgehört hast mit Firefox</h2>
+      <p class="wnp-main-tagline">
+        Deine offenen Tabs, Lesezeichen und Lieblingsseiten auf all deinen Geräten. Hol dir Firefox fürs Handy oder Tablet.
+      </p>
+    </div>
+    <div class="wnp-main-cta">
+      <h3>Scanne den QR&#x2011;code los geht's</h3>
+      {{ qrcode("https://app.adjust.com/xbd7d33_7sv2j5x?campaign=wnp107&adgroup=de") }}
```
[the remark this answers is not in the document]
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx107-de.html:31]: That's a fairly significant issue, new links requested!

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx107-de.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+<div class="wrapper">
+  <div class="wnp-main-content mzp-l-content mzp-t-content-xl mzp-l-columns mzp-t-columns-two">
+    <div>
+      <h2 class="wnp-main-title">Mach da weiter, wo du aufgehört hast mit Firefox</h2>
+      <p class="wnp-main-tagline">
+        Deine offenen Tabs, Lesezeichen und Lieblingsseiten auf all deinen Geräten. Hol dir Firefox fürs Handy oder Tablet.
+      </p>
+    </div>
+    <div class="wnp-main-cta">
+      <h3>Scanne den QR&#x2011;code los geht's</h3>
```
[the remark this answers is not in the document]
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx107-de.html:30]: It's specifically a non-breaking hyphen, not sure I can do that without the character code. But I could place a `<br>` tag after `den` to ensure `QR-code` stays on the same line.

**Comment by maureenlholland** [comment]
Please squash and merge if approved

**Review by craigcook — APPROVED** [review]
🥖 🌭

**Pull request merged**